OCP/ch9/1802-Haoran/src/PathOperations.java

```diff
@@ -0,0 +1,38 @@
+import java.io.IOException;
+import java.nio.file.FileVisitOption;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class PathOperations {
+    public static void cleanPath(Path path){
```

Comment:
I find the name a bit non descriptive but that might be a personal thing.
I think cleanDirectory might be more descriptive